streams/us_local.m3u

```diff
@@ -248,12 +232,8 @@ https://2-fss-1.streamhoster.com/pl_122/200562-1176456-1/playlist.m3u8
 https://reflect-golden-co.cablecast.tv/live-3/live/live.m3u8
 #EXTINF:-1 tvg-id="GreensboroTV.us",Greensboro TV (Greensboro NC) (720p)
 https://lin12.isilive.ca/live/greensboro/GTN/index.m3u8
-#EXTINF:-1 tvg-id="WTXLDT3.us",Grit Tallahassee (480p) [Not 24/7]
-https://content.uplynk.com/channel/c66d1d2d9e2c46cea1cf22a33f6ca488.m3u8
```

Comment:
works

Reply:
That is for [WTXL-DT1, ABC Tallahassee](https://github.com/iptv-org/iptv/pull/15484/files/85f3741832c4b7cd630e45fb1375541d82c9aa2f#diff-6bd195e23caaa57313e2bc2ee0c0c12bc5ae2d0217e1ea1514b70a5dfe8b55ffR33).